sqlglot/parser.py

```diff
@@ -558,6 +558,38 @@ class Parser(metaclass=_Parser):
         TokenType.PROPERTIES: lambda self: self._parse_wrapped_csv(self._parse_property),
     }
 
+    PROPERTY_PARSERS_TEXT = {
```

Comment:
can you just add this to property_parsers? i don't like how there's another dict

Reply:
Fixed in #1068